docs/onnx_custom_op/CMakeLists.txt

```diff
@@ -7,11 +7,12 @@ set(CMAKE_CXX_STANDARD 11)
 
 set(TARGET_NAME "onnx_custom_op")
 
-find_package(ngraph REQUIRED COMPONENTS onnx_importer)
+find_package(ngraph REQUIRED COMPONENTS onnx_ngraph_frontend)
```

Comment:
Maybe can we simplify the name? `onnx_frontend`?

Reply:
@nosovmik The same question for PDPD frontend.

Reply:
Frontends are discovered by FrontEndManager using OV_FRONTEND_PATH environment variable. FE Manager searches for "*_ngraph_frontend" libraries on this location and tries to load them. Mask "*_frontend" looked too generic for such search

Reply:
_ov_frontend?

Reply:
Yes, `*_ov_frontend` looks also fine. If you think this is more suitable, let's create ticket and correct this. It is better to do under separate PR as it will require updates of 'mock test' frontend names as well

Reply:
Can we rename to `ov_frontend`?

Reply:
It will need renaming of other frontends (like paddlepaddle and several frontends for testing purposes). Created ticket 61780 for that